doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -566,78 +566,104 @@ The query returns all the entities that match all the matching conditions (AND l
 (whose format depends on the operator), e.g.:
 
 ```
 temperature==50
 temperature<=20
 ```
 
+The syntax of an attribute path consists of a list of tokens separated by the `.` character. This list of tokens
+addresses a JSON property name, in accordance with the following rules:
+
+* The first token is the name of an NGSI attribute (*target NGSI attribute*) of an entity.
+* If filtering by attribute value (i.e. the expression is used in a `q` query), the rest of tokens (if present)
+  represent the path to a sub-property of the *target NGSI attribute value* (which should be a JSON object).
+  Such sub-property is defined as the *target property*.
+* If filtering by metadata (i.e. the expression is used in a `q` query), the second token represents a metadata
+  name associated to the target NGSI attribute, *target metadata*, and the rest of tokens
+  (if present) represent the path to a sub-property of the *target metadata value* (which should be a
+  JSON object). Such sub-property is defined as the *target property*.
+
+The *target property value* is defined as the value of the JSON property addressed by the list of tokens described
+above i.e. the value of the *target property*.
+
+In case only one token is provided (two in the case of filtering by metadata), then the *target property* will
+be the target NGSI attribute itself (or the target metadata in the case of filtering by metadata) and the
+*target property value* will be the target NGSI attribute value (or the target metadata value in the case
+of filtering by metadata). It is assumed that the value of the target NGSI attribute (or the target metadata
+in the case of filtering by metadata) is not a JSON object in this case.
+
+In the case some of the tokens include `.`, you can use simple quote (`'`) as separator. For example, the following
+attribute path `'a.b'.w.'x.y'` is composed by three tokens: the first token is `a.b`, the second token is `w` and
+the third token is `x.y`.
+
 The list of operators (and the format of the values they use) is as follows:
 
 + **Equal**: `==`. This operator accepts the following types of right-hand side:
-    + Single element, e.g. `temperature==40`. For an entity to match, it must contain an attribute
-      with that exact name (temperature) and value (40).
+    + Single element, e.g. `temperature==40`. For an entity to match, it must contain the target
+      property (temperature) and its value be the query value (40) (or include the value, in the case
+      the target property value is an array).
     + A list of comma-separated values, e.g. `color==black,red`. For an entity to match, it must
-      contain an attribute with that name and the value of the attribute must be **any** of the
-      values in the list (OR clause).
+      contain the target property and its value must be **any** of the values in the list (OR clause)
+      (or include **any** of the values in the list in the case the target property value is an array).
```

Comment:
in the case => in case

'the' should not be there. Only in very specific constructions, like "In the case of <NAME>, he/she ..."
This is how I 'feel' it, but, I'm not an English expert ...